wp-content/themes/darklight-site/sass/_layout/_global.scss

```diff
@@ -3,9 +3,50 @@
 }
 
 html {
-  font-size: 62.5%
+  font-size: 62.5%;
+  background-color: #212121;
+}
+
+h1 {
+  font-size: 4rem;
 }
 
 body {
   @include font-size(1.8);
+  font-family: helvetica;
+  background: #212121;
+  width: 96%;
+  margin: auto;
+}
+
+.banner {
+  background: #ffffff;
+}
+
+.body {
+  background: white;
+}
+
+.head {
+  text-align: center;
+  padding-top: 10px;
+
+}
+p {
+  @include font-size(2);
+}
+
+li {
+  display:inline;
+}
+
+.header__link-item {
+  margin: 10px;
+  color: black;
+  padding-top: 5px;
+  @include font-size(3);
+}
+
+.head a:hover {
```

Comment:
because sass is super awesome, you could combine this with the above
```
.head (changed to something else though) {
  &__link-item {
  }
 
 a:hover {
  }
}

```

the & extends the root name, ie head and will output `head`, `head__link-item` and `head a:hover`